luci-theme-ovh/htdocs/luci-static/resources/uikit.min.css

```diff
@@ -0,0 +1 @@
flexbox;display:flex;height:100%;-ms-flex-align:center;align-items:center}.guide-navbar__title{transition-delay:.2s;transition-property:left;position:absolute;display:-ms-flexbox;display:flex;-ms-flex-align:center;align-items:center;top:50%;left:3.2rem;transform:translateY(-50%);font-weight:600;font-size:1.75rem}.guide-navbar__title-small,.guide-navbar__title-sub,.guide-navbar__title-test{margin-bottom:-.1rem}.guide-navbar__title-text{margin-left:.7rem}.guide-navbar__title-small{font-weight:300;font-size:1.75rem;margin-left:.4rem}.guide-navbar__title-sub{font-weight:300;font-size:1.125rem;margin-left:.4rem;bottom:-.22rem}.guide-section{display:-ms-flexbox;display:flex;-ms-flex-direction:column;flex-direction:column;-ms-flex:0 0 auto;flex:0 0 auto;font-size:1rem;letter-spacing:.007rem;color:#122844}.guide-section__title{font-size:2rem;font-weight:300;letter-spacing:-.015rem}.guide-section__subtitle{font-size:1.2rem;font-weight:400;margin-bottom:1rem}.guide-section__article{padding:2rem 1rem}.guide-section__example{padding:1rem}.guide-section__example_empty{padding:0}.guide-section:last-child .guide-section__article{border-bottom:0}@media (min-width:768px) and (max-width:1679px){.guide-section{border-bottom-style:solid;border-bottom-width:1px;border-bottom-color:#eaece8}.guide-section:last-child{border-bottom:0}}@media (min-width:768px){.guide-navbar{position:relative}.guide-navbar__toggle{opacity:0;visibility:hidden;transition-delay:.2s}.guide-navbar__title{left:1rem;transition-delay:.4s}.guide-section__article{padding:2rem;border-bottom:0}.guide-section__example:not{padding:0 2rem 2rem}.guide-section__title{font-size:2.375rem}.guide-section__subtitle{font-size:1.5rem}}@media (min-width:1680px){.guide-section__article,.guide-section__example{padding:2rem;border-bottom-style:solid;border-bottom-width:1px}.guide-section{-ms-flex-direction:row;flex-direction:row}.guide-section:last-child{-ms-flex:1 0 auto;flex:1 0 auto}.guide-section__article{border-bottom-color:#eaece8}.guide-section__article,.guide-section__example{display:block;width:50%}.guide-section__example{font-weight:300;color:#fff;background-color:#273a55;border-bottom-color:#122844}}.guide-snippet{background-color:#122844;padding:1rem;margin:0 -1rem;font-size:.875rem;font-weight:500;color:#fff}.guide-snippet__comment{color:#999}.guide-snippet__keyword{color:#c792c2}.guide-snippet__string{color:#9cae64}.guide-snippet_html .guide-snippet__tag{color:#89ddfb}.guide-snippet_html .guide-snippet__name{color:#e07178}.guide-snippet_html .guide-snippet__string{color:#9cae64}.guide-snippet_html .guide-snippet__attr{color:#eecb6b}.guide-snippet_scss .guide-snippet__attribute{color:#b2cccf}.guide-snippet_scss .guide-snippet__selector-tag{color:#f78a58}.guide-snippet_scss .guide-snippet__number{color:#e07178}.guide-snippet_scss .guide-snippet__selector-class{color:#eecb6b}.guide-snippet_javascript .guide-snippet__params{color:#f78a58}@media (min-width:768px){.guide-snippet{padding:2rem;margin:0 0 1rem}}
```

Comment:
same in lato-heavyitalic-demo.html
http://ajax.googleapis.com/ajax/libs/jquery/1.7.2/jquery.min.js